pyproject.toml

```diff
@@ -0,0 +1,6 @@
+[tool.black]
+line-length = 120
```

Comment:
I know this is bikeshedding, but maybe we should stick to either 79 or 88; those two values seem to be the most common standards. 

https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#line-length motivation seems reasonable to me.

Reply:
I have a personal preference of 79, but I gave up imposing it in shared repositories a long time ago, so +1 to any choice.

And I am not sure 79 and 88 are the most common, PyCharm is probably the most popular Python IDE, and it enforces 120 by default, which probably has an effect on users.